src/DIRAC/RequestManagementSystem/Client/Operation.py

```diff
@@ -222,7 +222,7 @@ def __len__(self):
     @property
     def sourceSEList(self):
         """helper property returning source SEs as a list"""
-        return self.SourceSE.split(",") if self.SourceSE else [""]
+        return self.SourceSE.split(",") if self.SourceSE else []
```

Comment:
Same for targetSE couple lines below?

Reply:
Sure, I was a bit dubious about changing that one (my specific worry being if someone had a `for x in targetSEList` loop somewhere relying on the default containing a single element), but changing it doesn't seem to break any of my tests.

Reply:
It seems some of the modules (CheckMigration) do rely on the "empty string" target list to effectively act as a wildcard... I've reverted it back to only changing the source default for now.

Reply:
I think it is just the tests expecting that for some reason, I will try to have a look later today.
Fine for me to merge this regardless, but having these two properties with the same default sounds saner to me.

Reply:
I adapted the test by explicitly setting targetSE